golang/lib/services/shared_directory.go

```diff
@@ -1,21 +1,50 @@
 package services
 
+import (
+	"github.com/kurtosis-tech/kurtosis-client/golang/lib/services/shared_file_object"
+	"github.com/palantir/stacktrace"
+	"os"
+	"path/filepath"
+)
+
+//TODO add documentation
 type SharedDirectory struct {
 	//The service directory absolute path in the container where this code is running
-	absFilepathOnThisContainer string
+	sharedDirectoryMountpathOnThisContainer string
 	//The service directory absolute path in the service container
-	absFilepathOnServiceContainer string
+	sharedDirectoryMountpathOnServiceContainer string
 }
 
 func NewSharedDirectory(absFilepathOnThisContainer string, absFilepathOnServiceContainer string) *SharedDirectory {
-	return &SharedDirectory{absFilepathOnThisContainer: absFilepathOnThisContainer, absFilepathOnServiceContainer: absFilepathOnServiceContainer}
+	return &SharedDirectory{sharedDirectoryMountpathOnThisContainer: absFilepathOnThisContainer, sharedDirectoryMountpathOnServiceContainer: absFilepathOnServiceContainer}
 }
 
-func (s SharedDirectory) GetAbsFilepathOnThisContainer() string {
-	return s.absFilepathOnThisContainer
+//TODO add documentation
+func (s SharedDirectory) GetSharedDirectoryMountpathOnThisContainer() string {
```

Comment:
For simplicity, you could change thi sto `GetMountpathOnThisContainer` (since `SharedDirectory` is already the object, so it'll read like `sharedDir.GetMountpointOnThisContainer`)